rules/windows/privilege_escalation_unusual_parentchild_relationship.toml

```diff
@@ -15,7 +15,7 @@ language = "eql"
 license = "Elastic License v2"
 name = "Unusual Parent-Child Relationship"
 references = [
-    "https://github.com/sbousseaden/Slides/blob/master/Hunting MindMaps/PNG/Windows Processes%20TH.map.png",
+    "https://github.com/sbousseaden/Slides/blob/master/Hunting%20MindMaps/PNG/Windows%20Processes%20TH.map.png",
```

Comment:
I think there _may_ be more instances of this

Reply:
just checked, this should be it

Reply:
Also checked, could not find any other occurrence